jeremy2918/tsseract
# Implement NextJS

# [NextJS](https://trello.com/c/ZXxAMlGc/18-implement-nextjs-in-the-project-to-run-the-client-in-a-server)

## Description
This branch adds the (NextJS)[https://nextjs.org/] required dependencies to run with it. And it removes all the React Static generation files that are not needed anymore by the app. This branch also makes de necessary structural changes for the NextJS app to work. It includes de necessary [README.md](README.md) changes.

## Testing Steps

- `git pull`
- `git checkout ...`
- `npm install`
- `npm run dev:server`
- Check that the app is running on `http://localhost:3000`
